src/PlutusCore/Assembler/EntryPoint.hs

```diff
@@ -60,6 +61,8 @@ outputFilePath :: O.Parser (Maybe OutputFilePath)
 outputFilePath =
   O.argument (Just . OutputFilePath <$> O.str) (O.metavar "OUTPUT" <> O.value Nothing <> O.help "The output file path: defaults to stdout")
 
+shrinking :: O.Parser Bool
+shrinking = O.switch (O.long "shrinking" <> O.short 's' <> O.help "Shrink the uplc before serializing (only affects the assemble command)")
```

Comment:
I think it would be useful to respect `--shrinking` flag on `run` and `eval` commands too.

Reply:
That makes sense. I'll get to work on that.